### DISCUSSION TITLE
What is the difference between transform_from and transform_to in the SE3 object. Which one should be used for direct transformation and inverse transformation

### DISCUSSION THREAD

**Discussion by nea98** [facebookresearch/theseus#599] [Q&A] [answered]

**Comment by fantaosha** [comment, marked as answer]
Hello,

Thanks for your interests in Theseus! For g in SE(3), `g.transform_from(x) = g @ x` while `g.transform_to(x) = g.inverse() @ x`.

Best,
-Taosha

**Reply by mhmukadam** [reply]
Also in the backend where these functions point to the `torchlie` lib, `transform_from` == `transform` and `transform_to` == `untransform`.

**Reply by nea98** [reply]
Thank you for the response. It helped
